### ISSUE TITLE
Is .rdlc still the current report format file? OR is it .xml now.

### ISSUE THREAD

**Issue by JTWWI** [MicrosoftDocs/dynamics-365-unified-operations-public#4562]
In step 7 it says customize the report design, but in reports I find in my project none are .rdlc files, they're all .xml.

This makes it hard to follow since what I see in my enviornment is different than what this page shows.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: f5f9ccf3-650b-c8b0-efb9-b9d8a290db47
* Version Independent ID: 89633a0b-a776-f717-5e46-ffec1b9aa91e
* Content: [Expand Application Suite report data sets - Finance & Operations | Dynamics 365](https://learn.microsoft.com/en-us/dynamics365/fin-ops-core/dev-itpro/analytics/expand-app-suite-report-data-sets)
* Content Source: [articles/fin-ops-core/dev-itpro/analytics/expand-app-suite-report-data-sets.md](https://github.com/MicrosoftDocs/Dynamics-365-Unified-Operations-Public/blob/main/articles/fin-ops-core/dev-itpro/analytics/expand-app-suite-report-data-sets.md)
* Service: **dynamics-365-cross-app**
* Product: ****
* Technology: ****
* GitHub Login: @RichdiMSFT
* Microsoft Alias: **richdi**

**Comment by bishalgoswami** [comment]
Bug 817001 has been logged to address this issue.
